app/main.py

```diff
@@ -1,8 +1,28 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+    for human in people:
+        each_person = Person(human["name"], human["age"])
+        person_list.append(each_person)
+        for key in human.keys():
+            if key == "wife" and human["wife"] is not None:
+                each_person.wife = human["wife"]
+            elif key == "husband" and human["husband"] is not None:
+                each_person.husband = human["husband"]
+            else:
+                continue
```

Comment:
Instead of explicitly checking for the presence of keys, use the `dict.get()` method. This approach simplifies the code and avoids potential `KeyError` exceptions, making it cleaner and more concise.